wordpress-mobile/WordPress-iOS
# [SIWA] Show Apple credentials request.

Fixes # n/a 
WPAuth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/115

This uses the WPAuth changes in the above PR to show the Apple ID credentials request.

To test:
Run the app in Xcode 11.
- Log out if necessary.
- Select Log In > Continue with Apple.
- The 'Sign In with Apple' process is presented until valid credentials are entered or 'Cancel' is selected.
- Since account creation is not yet implemented, for now the user's email and full name will be logged in the console. Ex:
```
Apple Authenticator credentials: Optional("useremail@gmail.com"), Optional(givenName: First familyName: Last )
```

![siwa_flow](https://user-images.githubusercontent.com/1816888/63219849-dd334e80-c137-11e9-8f2b-2ddada766b08.png)


Run the app in Xcode 10. Verify the app runs and does not show the 'Continue with Apple' login button.

Update release notes:
